src/EnergyPlus/FileSystem.cc

```diff
@@ -404,7 +404,6 @@ namespace FileSystem {
         switch (ext) {
         case FileTypes::EpJSON:
         case FileTypes::JSON:
-        case FileTypes::GLHE:
```

Comment:
Yeah, I think the GLHE should still just be JSON.  Or if it needs to be changed, we probably shouldn't change that with just a couple days before code freeze.  I don't know, was this discussed?

Reply:
I discussed it briefly with @mitchute, but I think we both didn't realize the `.glhe` file was part of the regression tool. Like I said in the other comment, I will revert this for now and create a new PR to convert from `.glhe` after release.